### PR TITLE
File explorer refactoring

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**
Start working on File Explorer

### CODE REVIEW

File: libs/media/src/lib/file/explorer/explorer.component.html [hunk 1]
```diff
@@ -1,73 +1,77 @@
...
-          </button>
-        </ng-container>
+        <!--
+          <ng-container *ngIf="activeDirectory.type !== 'directory' && activeDirectory.multiple">
+            <button mat-stroked-button color="primary" (click)="openDialog()">
+              <mat-icon svgIcon="add"></mat-icon>
+              <span>Upload file</span>
+            </button>
+          </ng-container>
+        -->
```
Comment (pldespaigne) [inline libs/media/src/lib/file/explorer/explorer.component.html:34]: did you forget to clean ?

File: libs/media/src/lib/file/explorer/explorer.component.ts [hunk 2]
```diff
@@ -1,213 +1,189 @@
...
+  //         // oldRef is not set if it's a new upload and therefore a new file is added
+  //         if (!mediasToUpload[mediaIndex].oldRef && typeof this.activeDirectory.hasFile === 'number') {
+  //           this.activeDirectory.hasFile++
+  //         }
+  //       }
+  //       this.mediaService.uploadMedias(mediasToUpload);
+  //     }
+  //     this.dialogSubscription.unsubscribe();
+  //   });
+  // }
```
Comment (pldespaigne) [inline libs/media/src/lib/file/explorer/explorer.component.ts:179]: need to be cleaned ?

**Review by pldespaigne — APPROVED** [review]

File: libs/media/src/lib/file/explorer/explorer.component.html (hunk 1 above)
Comment (GrandSchtroumpf) [reply libs/media/src/lib/file/explorer/explorer.component.html:34]: No I'll put it back

File: libs/media/src/lib/file/explorer/explorer.component.ts (hunk 2 above)
Comment (GrandSchtroumpf) [reply libs/media/src/lib/file/explorer/explorer.component.ts:179]: I keep it as a reference for now

**Pull request merged**